charts/fluid-dataloader/alluxio/templates/job.yaml

```diff
@@ -10,6 +10,7 @@
   {{/*  {{- else -}}*/}}
   {{/*    {{- printf "Illegal release name. Should be like <dataset-name>-load-<suffix-length-5>. Current name: %s" .Release.Name | fail -}}*/}}
   {{/*  {{- end }}*/}}
+{{- if or (eq (lower .Values.dataloader.policy) "none") (eq (lower .Values.dataloader.policy) "once") }}
```

Comment:
I think we should also check `(eq (lower .Values.dataloader.policy) "")`, because it is possible that users may upgrading Fluid's dataset controller without upgrading DataLoad CRDs. In that case, dataloader.policy equals to `""` and we should make it work as usual

Reply:
Also, I think we can omit check `(eq (lower .Values.dataloader.policy) "none")` because from my understanding it's impossible to get this value.

Reply:
done. use '' to replae "none".